### PR TITLE
Bug 2107213: add warning to 2way replication about possible data-loss

### PR DESCRIPTION
**Pull request by weirdwiz**
image

### CODE REVIEW

File: packages/ocs/block-pool/body.tsx [hunk 1]
```diff
@@ -247,6 +248,15 @@ export const BlockPoolBody = (props: BlockPoolBodyPros) => {
...
             />
+            {state.replicaSize === '2' ? (
+              <Alert
+                title={t(
+                  'Data loss may occur, only recommended for small clusters or when backups are available or data loss is acceptable'
+                )}
+                variant={AlertVariant.info}
+                isInline
+              />
+            ) : null}
```
Comment (SanjalKatiyar) [inline packages/ocs/block-pool/body.tsx:259]:
```suggestion
            {state.replicaSize === '2' &&
              <Alert
                title={t(
                  'Data loss may occur, only recommended for small clusters or when backups are available or data loss is acceptable'
                )}
                variant={AlertVariant.info}
                isInline
              />
           }
```

**Comment by SanjalKatiyar** [comment]
is this change reviewed by UX ??

File: packages/ocs/block-pool/body.tsx [hunk 2]
```diff
@@ -247,6 +248,15 @@ export const BlockPoolBody = (props: BlockPoolBodyPros) => {
               onSelect={() => setReplicaOpen(false)}
               id="pool-replica-size"
             />
+            {state.replicaSize === '2' ? (
+              <Alert
+                title={t(
+                  'Data loss may occur, only recommended for small clusters or when backups are available or data loss is acceptable'
+                )}
```
Comment (SanjalKatiyar) [inline packages/ocs/block-pool/body.tsx:255]: run `yarn i18n`

**Comment by weirdwiz** [comment]
Change suggested by UX, making it draft.

**Comment by weirdwiz** [comment]
updated and made ready for review.
PTAL @SanjalKatiyar @bipuladh

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**